### DISCUSSION TITLE
Collection of Items in PO & Search Incomplete Outputs

### DISCUSSION THREAD

**Discussion by palagap** [inventree/InvenTree#3624] [Ideas]
Hi there, 

I have a couple of questions on available InvenTree features:

1. When adding line items to a purchase order, is it possible to specify that an order is for a collection of items rather than just one item? For example, if I am ordering two boxes that each contain 100 of a particular part and the purchase price per box is $50, then I could enter the unit price as $50 and the quantity as two. However, that would create an issue as the inventory should record 200 of the item at a unit price of $100/200 = $0.5. Is there a way to avoid having to do this calculation each time and instead specify that the order is for a collection of items rather than a single item?

<img width="1239" alt="Screen Shot 2022-08-31 at 1 55 10 PM" src="https://user-images.githubusercontent.com/40808444/187788933-a7f3d0ea-a90e-47f2-b986-502c93cafc3d.png">

2. When in the process of making a build order, is it possible to have a search bar to find parts in the incomplete outputs tab? It would be helpful in case there are many parts listed. This search bar is present in completed outputs. 

<img width="1656" alt="Screen Shot 2022-08-31 at 2 13 23 PM" src="https://user-images.githubusercontent.com/40808444/187788943-5136ef85-4df7-4588-9854-d90cac095c24.png">

Vs.

<img width="1678" alt="Screen Shot 2022-08-31 at 2 13 55 PM" src="https://user-images.githubusercontent.com/40808444/187788959-757f2a64-b060-4f0c-ad9c-bc2089f041cb.png">

Thanks.

**Comment by SchrodingersGat** [comment]
Regarding your first request - this has already been suggested an is an open issue - https://github.com/inventree/InvenTree/issues/1192

Regarding searching build outputs - this is a reasonable request, please open an "issue" so we can track this.

**Reply by palagap** [reply]
Great, thanks for the information - I'll make an issue for the second component.
